schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE animals(
+    id INT,
```

Comment:
- [ ] [OPTIONAL] Kindly edit the `id` to look like the following:
```
id int NOT NULL IDENTITY(1,1)
```
which makes the `id` column not accepting to NULL and makes it an [identity](https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-identity-column/).